### PR TITLE
Celestials Strands Plug-in

### PR DESCRIPTION
**Pull request by Saugia**
https://github.com/Saugia/celestial-strands

### CODE REVIEW

File: manifests/celestial-strands [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: Celestial Strands
+authors: Saugia
+homepage: https://github.com/Saugia/celestial-strands
+license: GPL-3.0-or-later
+version: v0.0.1
+shortDescription: Celestial Strands is a plug-in for Endless Sky that adds a new area of content based on the original Escape Velocity games, primarily Escape Velocity Override.
+description: Celestial Strands brings content from the original Escape Velocity series, primarily Escape Velocity Override, to Endless Sky in a new playable area. The plug-in focuses on the various factions of the Crescent, including the Strands, Zachit, and Miranu, and their presence in a new, young galaxy. Will the Strands reunite once more under the same banner, or will inner turmoil be their undoing yet again?
+url: https://github.com/Saugia/celestial-strands/archive/refs/tags/v0.0.1-alpha.zip
+iconUrl: https://github.com/Saugia/celestial-strands/blob/v0.0.1-alpha/icon.png
```
Comment (MCOfficer) [inline manifests/celestial-strands:9]:
```suggestion
iconUrl: https://github.com/Saugia/celestial-strands/raw/v0.0.1-alpha/icon.png
```

File: manifests/celestial-strands [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+autoupdate:
+  type: tag
+  url: https://github.com/Saugia/celestial-strands/archive/refs/tags/$version.zip
+  iconUrl: https://github.com/Saugia/celestial-strands/blob/$version/icon.png
```
Comment (MCOfficer) [inline manifests/celestial-strands:13]:
```suggestion
  iconUrl: https://github.com/Saugia/celestial-strands/raw/$version/icon.png
```

File: manifests/celestial-strands [hunk 3]
```diff
@@ -0,0 +1,13 @@
+name: Celestial Strands
+authors: Saugia
+homepage: https://github.com/Saugia/celestial-strands
+license: GPL-3.0-or-later
+version: v0.0.1
```
Comment (MCOfficer) [inline manifests/celestial-strands:5]: Should be your latest tag name, so
```suggestion
version: v0.0.1-alpha
```

File: manifests/celestial-strands [hunk 4]
```diff
@@ -0,0 +1,13 @@
+name: Celestial Strands
+authors: Saugia
+homepage: https://github.com/Saugia/celestial-strands
+license: GPL-3.0-or-later
```
Comment (MCOfficer) [inline manifests/celestial-strands:4]: You use the AGPL (GNU "Affero" GPL)

```suggestion
license: AGPL-3.0-or-later
```

**Review by MCOfficer — CHANGES_REQUESTED** [review]
Gotta change the filename to .yaml, so it gets picked up, well, everything

File: manifests/celestial-strands (hunk 4 above)
Comment (Saugia) [reply manifests/celestial-strands:4]: I swapped my LICENSE file to GPL, as that is what I intended. Does that work?

File: generated/PLUGINS.md [hunk 5]
```diff
@@ -9,6 +9,7 @@
 | ![Icon](https://raw.githubusercontent.com/ZBok/BetterVanillaPirates/v1.4/icon.png) | [Better Vanilla Pirates](https://github.com/ZBok/BetterVanillaPirates/archive/v1.4.zip) | ZBok | Makes pirates more interesting, varied, with many more pirate fighting missions. |
 | ![Icon](https://raw.githubusercontent.com/ZBok/BetterVanillaHai/0.9.2/icon.png) | [BetterVanillaHai](https://github.com/ZBok/BetterVanillaHai/archive/0.9.2.zip) | ZBok | Adds Hai ships with a wide variety of human/Hai outfits to human fleets in the late-game. Spices up pirate combat and adds variety. |
 | ![Icon](https://raw.githubusercontent.com/mOctave/blended-ships/v0.2.0/icon.png) | [Blended Ships](https://github.com/mOctave/blended-ships/archive/refs/tags/v0.2.0.zip) | mOctave | A spliced ships plugin for Endless Sky where all the ships are made in Blender. |
+| ![Icon](https://github.com/Saugia/celestial-strands/raw/v0.0.1-alpha/icon.png) | [Celestial Strands](https://github.com/Saugia/celestial-strands/archive/refs/tags/v0.0.1-alpha.zip) | Saugia | Celestial Strands is a plug-in for Endless Sky that adds a new area of content based on the original Escape Velocity games, primarily Escape Velocity Override. |
```
Comment (Saugia) [inline generated/PLUGINS.md:12]:
```suggestion
| ![Icon](https://github.com/Saugia/celestial-strands/raw/v0.0.1-alpha/icon.png) | [Celestial Strands](https://github.com/Saugia/celestial-strands/archive/refs/tags/v0.0.1b-alpha.zip) | Saugia | Celestial Strands is a plug-in for Endless Sky that adds a new area of content based on the original Escape Velocity games, primarily Escape Velocity Override. |
```

File: generated/plugins.json [hunk 6]
```diff
@@ -299,6 +299,22 @@
       "url": "https://github.com/Autohummer/Endless-Sky-Starbridge-Pack/archive/refs/tags/$version.zip"
     }
   },
+  {
+    "name": "Celestial Strands",
+    "authors": "Saugia",
+    "homepage": "https://github.com/Saugia/celestial-strands",
+    "license": "GPL-3.0-or-later",
+    "version": "v0.0.1-alpha",
```
Comment (Saugia) [inline generated/plugins.json:307]:
```suggestion
    "version": "v0.0.1b-alpha",
```

File: generated/plugins.json [hunk 7]
```diff
@@ -299,6 +299,22 @@
...
+    "shortDescription": "Celestial Strands is a plug-in for Endless Sky that adds a new area of content based on the original Escape Velocity games, primarily Escape Velocity Override.",
+    "description": "Celestial Strands brings content from the original Escape Velocity series, primarily Escape Velocity Override, to Endless Sky in a new playable area. The plug-in focuses on the various factions of the Crescent, including the Strands, Zachit, and Miranu, and their presence in a new, young galaxy. Will the Strands reunite once more under the same banner, or will inner turmoil be their undoing yet again?",
+    "url": "https://github.com/Saugia/celestial-strands/archive/refs/tags/v0.0.1-alpha.zip",
```
Comment (Saugia) [inline generated/plugins.json:310]:
```suggestion
    "url": "https://github.com/Saugia/celestial-strands/archive/refs/tags/v0.0.1b-alpha.zip",
```

File: generated/plugins.json [hunk 8]
```diff
@@ -299,6 +299,22 @@
...
+    "iconUrl": "https://github.com/Saugia/celestial-strands/raw/v0.0.1-alpha/icon.png",
```
Comment (Saugia) [inline generated/plugins.json:311]:
```suggestion
    "iconUrl": "https://github.com/Saugia/celestial-strands/raw/v0.0.1b-alpha/icon.png",
```

File: generated/plugins.yaml [hunk 9]
```diff
@@ -310,6 +310,26 @@
   autoupdate:
     type: tag
     url: https://github.com/Autohummer/Endless-Sky-Starbridge-Pack/archive/refs/tags/$version.zip
+- name: Celestial Strands
+  authors: Saugia
+  homepage: https://github.com/Saugia/celestial-strands
+  license: GPL-3.0-or-later
+  version: v0.0.1-alpha
```
Comment (Saugia) [inline generated/plugins.yaml:317]:
```suggestion
  version: v0.0.1b-alpha
```

File: generated/plugins.yaml [hunk 10]
```diff
@@ -310,6 +310,26 @@
...
+    area of content based on the original Escape Velocity games, primarily Escape
+    Velocity Override.
+  description: Celestial Strands brings content from the original Escape Velocity
+    series, primarily Escape Velocity Override, to Endless Sky in a new playable area.
+    The plug-in focuses on the various factions of the Crescent, including the Strands,
+    Zachit, and Miranu, and their presence in a new, young galaxy. Will the Strands
+    reunite once more under the same banner, or will inner turmoil be their undoing
+    yet again?
+  url: https://github.com/Saugia/celestial-strands/archive/refs/tags/v0.0.1-alpha.zip
+  iconUrl: https://github.com/Saugia/celestial-strands/raw/v0.0.1-alpha/icon.png
```
Comment (Saugia) [inline generated/plugins.yaml:328]:
```suggestion
  iconUrl: https://github.com/Saugia/celestial-strands/raw/v0.0.1b-alpha/icon.png
```

File: generated/plugins.yaml [hunk 11]
```diff
@@ -310,6 +310,26 @@
...
+  shortDescription: Celestial Strands is a plug-in for Endless Sky that adds a new
+    area of content based on the original Escape Velocity games, primarily Escape
+    Velocity Override.
+  description: Celestial Strands brings content from the original Escape Velocity
+    series, primarily Escape Velocity Override, to Endless Sky in a new playable area.
+    The plug-in focuses on the various factions of the Crescent, including the Strands,
+    Zachit, and Miranu, and their presence in a new, young galaxy. Will the Strands
+    reunite once more under the same banner, or will inner turmoil be their undoing
+    yet again?
+  url: https://github.com/Saugia/celestial-strands/archive/refs/tags/v0.0.1-alpha.zip
```
Comment (Saugia) [inline generated/plugins.yaml:327]:
```suggestion
  url: https://github.com/Saugia/celestial-strands/archive/refs/tags/v0.0.1b-alpha.zip
```

File: manifests/celestial-strands [hunk 12]
```diff
@@ -0,0 +1,13 @@
...
+version: v0.0.1-alpha
```
Comment (Saugia) [inline manifests/celestial-strands:5]:
```suggestion
version: v0.0.1b-alpha
```

File: manifests/celestial-strands [hunk 13]
```diff
@@ -0,0 +1,13 @@
...
+shortDescription: Celestial Strands is a plug-in for Endless Sky that adds a new area of content based on the original Escape Velocity games, primarily Escape Velocity Override.
+description: Celestial Strands brings content from the original Escape Velocity series, primarily Escape Velocity Override, to Endless Sky in a new playable area. The plug-in focuses on the various factions of the Crescent, including the Strands, Zachit, and Miranu, and their presence in a new, young galaxy. Will the Strands reunite once more under the same banner, or will inner turmoil be their undoing yet again?
+url: https://github.com/Saugia/celestial-strands/archive/refs/tags/v0.0.1-alpha.zip
+iconUrl: https://github.com/Saugia/celestial-strands/raw/v0.0.1-alpha/icon.png
```
Comment (Saugia) [inline manifests/celestial-strands:9]:
```suggestion
iconUrl: https://github.com/Saugia/celestial-strands/raw/v0.0.1b-alpha/icon.png
```

**Review by Saugia — COMMENTED** [review]
Updating release tag. 
https://github.com/Saugia/Celestial-Strands/releases/tag/v0.0.1b-alpha

File: generated/PLUGINS.md [hunk 14]
```diff
@@ -9,6 +9,7 @@
 | ![Icon](https://raw.githubusercontent.com/ZBok/BetterVanillaPirates/v1.4/icon.png) | [Better Vanilla Pirates](https://github.com/ZBok/BetterVanillaPirates/archive/v1.4.zip) | ZBok | Makes pirates more interesting, varied, with many more pirate fighting missions. |
 | ![Icon](https://raw.githubusercontent.com/ZBok/BetterVanillaHai/0.9.2/icon.png) | [BetterVanillaHai](https://github.com/ZBok/BetterVanillaHai/archive/0.9.2.zip) | ZBok | Adds Hai ships with a wide variety of human/Hai outfits to human fleets in the late-game. Spices up pirate combat and adds variety. |
 | ![Icon](https://raw.githubusercontent.com/mOctave/blended-ships/v0.2.0/icon.png) | [Blended Ships](https://github.com/mOctave/blended-ships/archive/refs/tags/v0.2.0.zip) | mOctave | A spliced ships plugin for Endless Sky where all the ships are made in Blender. |
+| ![Icon](https://github.com/Saugia/celestial-strands/raw/v0.0.1b-alpha/icon.png) | [Celestial Strands](https://github.com/Saugia/celestial-strands/archive/refs/tags/v0.0.1-alpha.zip) | Saugia | Celestial Strands is a plug-in for Endless Sky that adds a new area of content based on the original Escape Velocity games, primarily Escape Velocity Override. |
```
Comment (Saugia) [inline generated/PLUGINS.md:12]:
```suggestion
| ![Icon](https://github.com/Saugia/celestial-strands/raw/v0.0.1b-alpha/icon.png) | [Celestial Strands](https://github.com/Saugia/celestial-strands/archive/refs/tags/v0.0.1b-alpha.zip) | Saugia | Celestial Strands is a plug-in for Endless Sky that adds a new area of content based on the original Escape Velocity games, primarily Escape Velocity Override. |
```

File: generated/plugins.yaml [hunk 15]
```diff
@@ -310,6 +310,26 @@
...
```
Comment (Saugia) [inline generated/plugins.yaml:327]:
```suggestion
  url: https://github.com/Saugia/celestial-strands/archive/refs/tags/v0.0.1b-alpha.zip
```

File: manifests/celestial-strands [hunk 16]
```diff
@@ -0,0 +1,13 @@
+name: Celestial Strands
+authors: Saugia
+homepage: https://github.com/Saugia/celestial-strands
+license: GPL-3.0-or-later
+version: v0.0.1b-alpha
+shortDescription: Celestial Strands is a plug-in for Endless Sky that adds a new area of content based on the original Escape Velocity games, primarily Escape Velocity Override.
+description: Celestial Strands brings content from the original Escape Velocity series, primarily Escape Velocity Override, to Endless Sky in a new playable area. The plug-in focuses on the various factions of the Crescent, including the Strands, Zachit, and Miranu, and their presence in a new, young galaxy. Will the Strands reunite once more under the same banner, or will inner turmoil be their undoing yet again?
+url: https://github.com/Saugia/celestial-strands/archive/refs/tags/v0.0.1-alpha.zip
```
Comment (Saugia) [inline manifests/celestial-strands:8]:
```suggestion
url: https://github.com/Saugia/celestial-strands/archive/refs/tags/v0.0.1b-alpha.zip
```

**Review by Saugia — COMMENTED** [review]
Again.

File: manifests/celestial-strands (hunk 4 above)
Comment (MCOfficer) [reply manifests/celestial-strands:4]: yep

**Pull request merged**